models/organization/src/team/types.ts

```diff
@@ -1,9 +1,16 @@
+import type { TReference } from '@commercetools-test-data/commons';
 import type { TBuilder } from '@commercetools-test-data/core';
 import type { TCoreTeam } from '@commercetools-test-data/graphql-types/src/generated/core';
 
 export type TTeam = Omit<TCoreTeam, '__typename'>;
 
 export type TTeamGraphql = TCoreTeam;
 
-export type TTeamBuilder = TBuilder<TTeam>;
+export type TTeamRest = {
```

Comment:
Nit: I would name this `TTeam` and the GraphQL representation `TTeamGraphql`. No other types should be needed.

Reply:
Oh, yes. These are great suggestions, applied in 2aeb66a05f07c24c1bc179e2bfae8c407fdcbd53